static/js/src/entry/user-portal/routes/home-exact/Index.vue

```diff
@@ -1,44 +1,194 @@
 <template>
-  <div>
-    <message :name="welcomeMessageKey"> <p>Welcome message.</p> </message>
-    <message :name="comingSoonMessageKey">
-      <p>Coming soon message.</p>
-    </message>
-    <ul>
-      <li v-for="datum in data" :key="datum.id">{{ datum.headline }}</li>
-    </ul>
+  <div class="has-white-bg-from-bp-l has-ump-top-padding">
+    <messages :num-messages="2">
+      <template v-slot="slotProps">
+        <message
+          heading="Welcome"
+          :name="welcomeMessageKey"
+          @setMessageSeen="slotProps.setMessageSeen"
+        >
+          <template v-slot:icon>
+            <icon name="camera" :display="{ size: 'm' }" />
+          </template>
+          <template v-slot:content>
+            <p class="has-text-gray-dark t-space-heading-m">
+              <em>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+                enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                nisi ut aliquip ex ea commodo consequat.
+              </em>
+            </p>
+          </template>
+        </message>
+
+        <message
+          heading="Coming soon"
+          :name="comingSoonMessageKey"
+          @setMessageSeen="slotProps.setMessageSeen"
+        >
+          <template v-slot:icon>
+            <icon name="camera" :display="{ size: 'm' }" />
+          </template>
+          <template v-slot:content>
+            <p class="has-text-gray-dark t-space-heading-m">
+              <em>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+                eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+                enim ad minim veniam, quis nostrud exercitation ullamco laboris
+                nisi ut aliquip ex ea commodo consequat.
+              </em>
+            </p>
+          </template>
+        </message>
+      </template>
+    </messages>
+
+    <h1 class="has-xxl-btm-marg has-ump-side-padding">Your Account</h1>
+
+    <div class="c-summary-boxes has-xl-btm-marg has-ump-side-padding">
+      <summary-box heading="contact info">
+        <template v-slot:content>
+          <info-list :items="data">
+            <template v-slot="slotProps">
+              <span
+                class="has-text-gray-dark"
+                :class="{ 't-wrap-break': slotProps.item.heading === 'Email' }"
+              >
+                {{ slotProps.item.text }}
+              </span>
+            </template>
+          </info-list>
+        </template>
+
+        <template v-slot:links>
+          <ul v-if="!pwResetSuccess && !pwResetFailure" class="c-link-list">
+            <li>
+              <span class="c-link-list__arrow has-text-teal">
+                <strong>&rarr;</strong>
```

Comment:
In the mockup it kinda looks like this is an actual icon as opposed to an HTML entity. Not a huge deal either way, just an aesthetic thing. Maybe there's a benefit to _not_ using an SVG?

Reply:
Shhhh ... I'm hoping to sneak in the HTML entity because I think it's better for accessibility + just easier overall.

Reply:
LOL, well I'd argue that being read "right arrow" probably doesn't do much for a nonsighted or vision impaired user. That's how I look at icons in general. They're usually just for visual presentation and deserve a text descriptor different from the icon itself. For instance no one wants to hear "three stacked bars" when using voiceover, JAWs etc; they want to hear "menu"